ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -213,7 +213,11 @@ then
 	if [ $BOARD_FMUV3 != 0 ]
 	then
 		# sensor heating is available, but we disable it for now
-		param set SENS_EN_THERMAL 0
+		param set SENS_EN_THERMAL 0        
+        
+		# probe for ICM20948 (Here GPS) on external I2C. Rotation 6 (270 deg yaw)
+		# for the accel and gyro and rotation 4 (180 deg yaw)
+		mpu9250 -X -M -R 6 start
```

Comment:
Why isn't this with the other external mags at the top?

Reply:
Not sure which section you're referring to, do you mean like that:

```
if ver hwcmp PX4FMU_V2
then
	# External I2C bus
	hmc5883 -C -T -X start
	lis3mdl -X start
	ist8310 -C start
        
    # probe for ICM20948 (Here GPS) on external I2C. Rotation 6 (270 deg yaw)
    # for the accel and gyro and rotation 4 (180 deg yaw)
    mpu9250 -X -M -R 6 start
```

Reply:
Yes there. Unless I'm missing some reason this should only be for the pixhawk 2 cube.

Reply:
Fixed.